packages/patternfly-4/react-core/src/components/ContextSelector/ContextSelector.tsx

```diff
@@ -1,69 +1,73 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import FocusTrap from 'focus-trap-react';
+import * as React from 'react';
 import styles from '@patternfly/patternfly/components/ContextSelector/context-selector.css';
```

Comment:
Change to `@patternfly/react-styles/css/components/ContextSelector/context-selector`